### ISSUE TITLE
Database engine per mapper/entity

### ISSUE THREAD

**Issue by JeromeDane** [vlucas/phpDataMapper#9]
Need a better way of setting table engine ("MyISAM", "InnoDB", etc.) through the mapper class instead of tying it to the adapter, which forces you to create multiple adapters if you want to use multiple engines within a database (which many applications do in order to take advantage of text indexing in MyISAM and the speed/stability of InnoDB..) 

There are probably issues I'm unaware of, but I know this is something I've run into. Feel free to comment/close/etc. Just wanted to bring it up.

**Comment by vlucas** [comment]
I totally agree this would be a great feature. I actually already want to use it in a current project that needs MyISAM's FULLTEXT searching capabilities.

**Comment by actridge** [comment]
I added this to Spot, the PHP 5.3 fork of phpDataMapper:
https://github.com/actridge/Spot/commit/43b10d9c4020faeea252d3fe63bd3267f3cde62a
